aggregator/src/main/java/com/redhat/cloud/notifications/db/EndpointEmailSubscriptionResources.java

```diff
@@ -0,0 +1,59 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import org.hibernate.Session;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+@ApplicationScoped
+public class EndpointEmailSubscriptionResources {
+
+    @Inject
+    Session session;
+
+    @Transactional
+    public Boolean subscribe(String accountNumber, String username, String bundleName, String applicationName, EmailSubscriptionType subscriptionType) {
```

Comment:
This method is only used by tests. It should not be part of the `main` sources.

Reply:
Good catch. I found this as well but didn't want to touch it as it's copied from notifications-backend and I thought we would get rid of it later when removing the duplicates.

I will move it for now.